src/Export.js

```diff
@@ -17,7 +17,7 @@ const Export = ({ API }) => {
       <SiApple /> WebCal (eg iOS)
     </Dropdown.Item>
     <Dropdown.Item eventKey="gcal" target={"_blank"} rel={"noreferrer"} 
-      href={`https://www.google.com/calendar/render?cid=${webcal}`}>
+      href={`https://www.google.com/calendar/render?cid=${encodeURIComponent(webcal.replace('https', 'http'))}`}>
```

Comment:
Dug a bit deeper and I'll need to refactor the link generation (to fix Exchange/Outlook), but I'll do that in another branch